lib/octokit/client/apps.rb

```diff
@@ -154,10 +158,12 @@ def remove_repository_from_integration_installation(installation, repo, options
       #
       # @see https://developer.github.com/apps/building-integrations/setting-up-and-registering-github-apps/identifying-users-for-github-apps/
       #
-      # @return [Array<Sawyer::Resource>] A list of repositories
+      # @return [Sawyer::Resource] A list of repositories
```

Comment:
Here, too.

I realize that these were already wrong, and you made them less wrong—but since this is being tweaked, I'd love to get them right.